hands-on-exercise/5-run-in-prod/4-genesis.md

```diff
@@ -31,11 +31,17 @@ The genesis only needs to be generated once, then it is distributed and copied o
 
 One of the simpler parameters that you can decide on is the name of the staking token. The name you choose is for the indivisible denomination. For the Cosmos Hub that is `uatom`, which is understood as a millionth of an ATOM.
 
-For instance, if you decide that your token is the STONE and that each STONE contains a billion indivisible units, by convention you would name the unit `nstone`, as in a _nano STONE_. The chain only cares about this `nstone` unit. The STONE will be an artifact reserved for human interaction.
+For instance, if you decide that your token is the STONE and that each STONE contains a billion indivisible units, by convention you would name the unit `nstone`, as in a _nano STONE_. The chain only cares about this `nstone` unit. The `STONE` will be an artifact reserved for human interaction.

```

Comment:
I did not put STONE with backquotes precisely because this is not a dev concept. Like we write ATOM, BTC or ETH.